signup.css

```diff
@@ -108,8 +108,14 @@
     border-radius: 3px;
     text-align: center;
 }
+.login{
+    padding: 0rem;
+}
+.forgot-password{
+    padding: 0rem;
+}
```

Comment:
is this needed? if yes then u don't need to give 0rem, 0 is enough and u merge the two

Reply:
Done